setup.py

```diff
@@ -105,13 +105,13 @@
     "grpcio~=1.43.0",
     "grpcio-tools~=1.43.0",
     "protobuf~=3.19.3",
-    "azure-functions==1.11.0"
+    "azure-functions==1.11.0",
+    "python-dateutil~=2.8.2",
```

Comment:
Do we need to pin it to 2.8.2 or should we just install the latest version?

Reply:
as discussed offline, dateutil is released infrequently, will keep the pinning for now